### DISCUSSION TITLE
Pinch to zoom and gesture doesnt work

### DISCUSSION THREAD

**Discussion by llyonard** [linuxdeepin/developer-center#3537] [Troubleshooting | 故障排查]
Hi, i have an Asus um325ua, with deepin 20.8 new installation(tried both kernel, the old one has even wifi problem on my machine), cant use common gesture like pinch to zoom and touchpad in general seems a bit unresponsive too. I miss something?

**Comment by BLumia** [comment]
Which application? Could you name a few examples? Or did you mean all applications have this issue?

**Comment by llyonard** [comment]
All application as far as i know, tried with different browser (default one, firefox from flatpak or binary etc), pretty sure is system wide.
